### ISSUE TITLE
More Info Screen - VoiceOver Crashes "About this App" Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#74]
When I go to the "About this App" screen in the "More Info" section using VoiceOver (screen reader) the application crashes.

**Comment by sanganalakshay** [comment]
@jfc3-dol  Can you tell me in which devices this issue persists? I tested on iPhone 6s and the VoiceOver was reading all the content in the "About this App" screen in the "More Info" section.

**Comment by jfc3-dol** [comment]
@sanganalakshay,

I have been using our iPad Air 2 which is using iOS 10.3.3 (model - MGLW2LL/A produced from 10/2014 - 03/2017 ) and crashed when you go to the "About This App" screen after a few seconds when VoiceOver is running.  Not sure why it's crashing since itI has four times the amount of RAM as the iPod Touch. It is the older version because it has been here for a few years.

The "About This App" screen does not crash on our iPod Touch (model - ME643LL/A  produced from 05/2013 - 06/2014) which is using iOS 9.3.5.

**Comment by carolynhuang** [comment]
It seems that this is an ipad issue, which cannot be resolved. Developers were unable to replicate on mobile devices.

**Comment by jfc3-dol** [comment]
It's no longer crashing the application when reading this screen with VoiceOver.

Closing issue.
